packages/gimbal/src/command/audit/index.ts

```diff
@@ -112,6 +102,67 @@ const audit = async (options: CommandOptions): Promise<Report> => {
     }
   }
 
+  return {
+    data: rets,
+    success,
+  };
+};
+
+const audit = async (options: CommandOptions): Promise<Report | Report[]> => {
+  // if we are going to calculate unused CSS, take heap snapshot(s) or run lighthouse audits
+  // we need to host the app and use chrome
+  const needChromeAndServe = options.calculateUnusedCss || options.heapSnapshot || options.lighthouse;
+
+  const servePort = needChromeAndServe ? await findPort() : null;
+  const buildDir = resolvePath(options.cwd, options.buildDir as string);
+  const serve = servePort ? new Serve({ port: servePort, public: buildDir }) : null;
+  const chrome = needChromeAndServe ? new Chrome() : undefined;
+
+  let report: Report | Report[];
+
+  if (serve) {
+    await serve.start();
+  }
+
+  if (chrome) {
+    await chrome.launch();
+  }
+
+  if (Array.isArray(options.route)) {
+    const queue = new Queue();
```

Comment:
Had to use a sequential queue, lighthouse threw errors trying to get it to run in parallel even with different chrome processes. Assume lighthouse internals are only setup to go with one audit at a time.